src/store/asana/index.ts

```diff
@@ -389,6 +390,7 @@ export const useAsanaStore = defineStore("asana", {
     },
 
     LOAD_STORIES(task: Task): void {
+      this.storiesLoading = true; // Set this before the action is invoked
       const loadStories = async () => {
         if (asanaClient) {
           const storiesResponse = await asanaClient.stories
```

Comment:
Do we need a tach here to set storiesLoading = false if there is an error?

Reply:
isn't it just 
```
const storiesResponse = await asanaClient.stories.catch(()=>{
    this.storiesLoading = false;
})

Reply:
Almost. Slightly different because the .catch() return will impact the type for storiesResponse. I've updated the code in this PR